### PR TITLE
 Add configuration option for packet chunk size

### PR DESCRIPTION
**Pull request by sebtomba**
## Overview
Add configuration option for packet chunk size

### JIRA ticket:
https://rchain.atlassian.net/browse/RCHAIN-3117

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### CODE REVIEW

**Comment by rabbitonweb** [comment]
Compilation failed

**Comment by sebtomba** [comment]
@rabbitonweb compilation passed. Please review.

File: comm/src/test/scala/coop/rchain/comm/transport/TcpTransportLayerSpec.scala [hunk 1]
```diff
@@ -50,7 +50,14 @@ class TcpTransportLayerSpec
   ): Task[(TransportLayer[Task], TransportLayerShutdown[Task])] =
     for {
       client <- CachedConnections[Task, TcpConnTag].map { implicit cache =>
-                 new GrpcTransportClient(env.cert, env.key, 4 * 1024 * 1024, tempFolder, 100)
+                 new GrpcTransportClient(
+                   env.cert,
+                   env.key,
+                   4 * 1024 * 1024,
```
Comment (rabbitonweb) [inline comm/src/test/scala/coop/rchain/comm/transport/TcpTransportLayerSpec.scala]: consider using "named parameters"

File: node/src/main/resources/reference.conf [hunk 2]
```diff
@@ -45,8 +45,11 @@ rnode {
     # Maximum number of peers allowed to connect to the node
     max-connections = 20
 
-    # Maximum size of message that can be sent via transport layer
-    max-message-size = 256K
+    # Maximum size of message that can be received via transport layer
+    max-message-size = 2M
```
Comment (rabbitonweb) [inline node/src/main/resources/reference.conf]: wowowow why 2M?

Comment (sebtomba) [reply node/src/main/resources/reference.conf]: @rabbitonweb Why 256K? :)
This is the max incoming message size a node will accept. Why would a node accept only messages that are smaller than 256K? The default gRPC max message size is 4MB. So 2MB is 50% of that and in my opinion a quite fair message size.

**Pull request merged**